eg/Classes/PluginInstall.py

```diff
@@ -283,3 +431,154 @@ def VisitName(self, node):
+            self,
+            -1,
+            size=(-1, 325),
+            style=wx.SUNKEN_BORDER,
+            agwStyle=(
+                customtreectrl.TR_AUTO_TOGGLE_CHILD |
+                customtreectrl.TR_AUTO_CHECK_PARENT |
+                customtreectrl.TR_TWIST_BUTTONS |
+                customtreectrl.TR_HAS_BUTTONS |
+                customtreectrl.TR_HAS_VARIABLE_ROW_HEIGHT |
+                customtreectrl.TR_FULL_ROW_HIGHLIGHT
+            )
+        )
+
+        buttonRow = eg.ButtonRow(self, (wx.ID_OK, wx.ID_CANCEL))
+
+        tree.EnableSelectionGradient(True)
+        tree.SetGradientStyle(False)
+        tree.SetFirstGradientColour(wx.Colour(0, 0, 0))
+        tree.SetSecondGradientColour(wx.Colour(0, 255, 0))
+        tree.SetBackgroundColour(
+            wx.SystemSettings_GetColour(wx.SYS_COLOUR_WINDOW)
+        )
+
+        imageList = wx.ImageList(16, 16)
+        imageList.Add(PilToBitmap(StreamToPil(CHECKED)))
+        imageList.Add(PilToBitmap(StreamToPil(UNCHECKED)))
+        tree.SetImageListCheck(16, 16, imageList)
+
+        sizer = wx.BoxSizer(wx.VERTICAL)
+        sizer.Add(headerBox, 0, wx.EXPAND | wx.BOTTOM, 20)
+        sizer.Add(tree, 0, wx.EXPAND | wx.ALIGN_CENTER | wx.ALL, 10)
+        sizer.Add(buttonRow.sizer, 0, wx.EXPAND)
+
+        self.SetSizer(sizer)
+        self.Bind(wx.EVT_CLOSE, self.OnCancel)
+
+        self.FillTree()
+        tree.Bind(customtreectrl.EVT_TREE_ITEM_CHECKED, self.OnItemCheck)
+
+    def ShowModal(self, *args, **kwargs):
+        self.SendSizeEvent()
+        wx.Dialog.ShowModal(self, *args, **kwargs)
+
+    def FillTree(self):
+
+        def ReadFileFolders(ff, parent):
+            for key in sorted(ff.keys()):
+                if key in ('flags', 'sourcePath'):
+                    continue
+                if isinstance(ff[key], dict):
+                    child = self.tree.AppendItem(
+                        parent,
+                        key,
+                        1,
+                        wx.StaticText(self.tree, -1, str(ff[key]['flags']))
+                    )
+
+                    self.tree.SetPyData(child, ff[key])
+                    ff[key]['flags'] = not bool(ff[key]['flags'])
+                    self.tree.CheckItem(child, ff[key]['flags'])
+
+                    ReadFileFolders(ff[key], child)
+                else:
+                    child = self.tree.AppendItem(
+                        parent,
+                        key,
+                        1,
+                        wx.StaticText(self.tree, -1, str(ff[key]))
+                    )
+
+                    ff[key] = not bool(ff[key])
+                    self.tree.SetPyData(child, ff[key])
+                    self.tree.CheckItem(child, ff[key])
+
+        self.root = root = self.tree.AddRoot(self.fileFolders['sourcePath'])
+        self.tree.SetPyData(root, self.fileFolders)
+        ReadFileFolders(self.fileFolders, root)
+        self.tree.Expand(root)
```

Comment:
And the thing with the gradient was something new. because I have not messed around with that specific wx widget before. I will change it to the stock look.